cpp/src/gandiva/function_registry_string.cc

```diff
@@ -209,6 +209,71 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
                      "convert_replace_invalid_fromUTF8_binary",
                      NativeFunction::kNeedsContext),
 
+      NativeFunction("convert_toDOUBLE", {"convert_todouble"}, DataTypeVector{float64()},
+                     binary(), kResultNullIfNull, "convert_toDOUBLE_binary",
+                     NativeFunction::kNeedsContext),
+
+      NativeFunction("convert_toDOUBLE_BE", {"convert_todouble_be"},
+                     DataTypeVector{float64()}, binary(), kResultNullIfNull,
+                     "convert_toDOUBLE_binary_be", NativeFunction::kNeedsContext),
+
+      NativeFunction("convert_toFLOAT", {"convert_tofloat"}, DataTypeVector{float32()},
+                     binary(), kResultNullIfNull, "convert_toFLOAT_binary",
```

Comment:
convert_toFLOAT_binary name seems ambigious..lets keep the function name as convert_toFLOAT

Reply:
I removed the binary suffix from the functions names